recipes/torchcam/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "torchcam" %}
+{% set version = "0.3.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchcam-{{ version }}.tar.gz
+  sha256: 400d451cbbafe0dbddbf3a22feab9e23d1317e90cf6eae16bca96e0e2ff4f076
+
+build:
+  number: 0
+  noarch: python
+  script: 
+    ## Show the rendered recipe
+    - echo -e "\n>>> Show recipe '{{ name }}/meta.yaml' <<<\n\n"
+    - echo -e $(cat $RECIPE_DIR/meta.yaml)  # [not win]
+    ## Ensure inclusion of requirements.txt
+    {% set reqfile = "requirements.txt" %}
+    {% set reqsource = name ~ ".egg-info/requires.txt" %}
+    # IF {{ reqfile }} is NOT PRESENT, COPY {{ reqfile }} from {{ reqsource }}
+    - echo -e "\n>>> ENSURE INCLUSION OF '{{ reqfile }}' <<<\n\n"
+    - if [ ! -f {{ reqfile }} ] && [ -f {{ reqsource }} ]; then cp {{ reqsource }} {{ reqfile }}; fi  # [not win]
+    #- copy {{ name }}.egg-info\requires.txt {{ reqfile }}  # [win]
+    - copy {{ reqsource | replace('/', '\\') }} {{ reqfile }}  # [win]
+    ## Ensure exclusion of tests folder
```

Comment:
The best way to do this is by changing it upstream. Should be fine for now but would be good to have a PR with a permanent fix.

Reply:
Changing upstream could take longer and sometimes there's no response. I did this in another PR recently: https://github.com/conda-forge/staged-recipes/pull/17275. The if conditionals will make sure of only creating the requirements.txt file when it is not present.

Reply:
I will like to clean up the recipe later (and remove anything extra). Including the conditionals are temporary measures only.

Reply:
> Changing upstream could take longer and sometimes there's no response.
Yes that's why I meant that it's fine for now but a PR would be good nonetheless.

> The if conditionals will make sure of only creating the requirements.txt file when it is not present.
It seems like it's currently available so it wouldn't be needed?

Reply:
Yes, you are right about that. Just removed the lines for "copying/creating requirements.txt" .